devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -74,12 +74,12 @@ metadata {
 
 	preferences {
 		input "motionDelayTime", "enum", title: "Motion Sensor Delay Time",
-			options: ["20 seconds", "40 seconds", "1 minute", "2 minutes", "3 minutes", "4 minutes"]
+			options: ["30 seconds", "40 seconds", "1 minute", "2 minutes", "3 minutes", "4 minutes"]
 
 		input "motionSensitivity", "enum", title: "Motion Sensor Sensitivity", options: ["maximum", "normal", "minimum", "disabled"]
 
 		input "reportInterval", "enum", title: "Report Interval", description: "How often the device should report in minutes",
-			options: ["8 minutes", "15 minutes", "30 minutes", "1 hour", "6 hours", "12 hours", "18 hours", "24 hours"]
+			options: ["1 minute", "2 minutes", "3 minutes", "4 minutes", "8 minutes", "15 minutes", "30 minutes", "1 hour", "6 hours", "12 hours"]
```

Comment:
we don't want to allow the interval this low

we find that customers get very concerned when their batteries die quickly

Reply:
Even when set to 1 minute, the lowest setting is 4 minutes on battery power, Wakeup interval generally sets the minimum most setting so it generally won't change unless wakeup interval is set to 240 seconds which is the lowest setting.

If Wakeup Interval = 3600
If Parameter 111 = 60

Sensors will still report every 60 seconds. 

Only on USB power is this setting unlocked to as low as once per a second (or in this case 1 minute) since wakeup does not play a hand in this anylonger as a repeater device.

Reply:
ok